tests/providers/test_adapter.py

```diff
@@ -79,6 +87,124 @@ def check_to_braket_openqasm_unitary_correct(qasm_program: Program | str):
 class TestAdapter(TestCase):
     """Tests adapter."""
 
+    def test_target(self):
```

Comment:
These belong in the adapter tests, since they don't make any reference to a backend whatsoever